tests/main/interfaces-system-observe/task.yaml

```diff
@@ -10,39 +13,66 @@ details: |
 
 prepare: |
     echo "Given a snap declaring a plug on the system-observe interface is installed"
-    snapbuild $TESTSLIB/snaps/system-observe-consumer .
-    snap install --dangerous system-observe-consumer_1.0_all.snap
+    snap install --edge test-snapd-system-observe-consumer
+
+    echo "And the D-bus X11 dependencies are installed"
+    apt install -y dbus-x11
+
+    if [[ "$SPREAD_SYSTEM" != ubuntu-14.04-* ]]; then
```

Comment:
The logic for all of these 14.04 tests looks fine, but I wonder if it would be better to set a variable, eg:

```
do_hostnamed_tests="yes"
if [[ "$SPREAD_SYSTEM" = ubuntu-14.04-* ]]; then
    do_hostnamed_test="no"
fi
...
if [ "$do_hostnamed_tests" = "yes" ]; then
...
fi
...
```

This way if you want to test for more than ubuntu-14.04-* you can make your changes more easily.